edx-platform/pearson-vue-theme/lms/templates/course_experience/course-dates-fragment.html

```diff
@@ -8,16 +8,11 @@
 from django.utils.translation import ugettext as _
 %>
 % if len(course_date_blocks) > 0:
-    <h3 class="hd hd-6 handouts-header">${_("Upcoming Dates")}</h3>
+    <h3 class="hd hd-6 handouts-header">${_("Upcoming dates")}</h3>
```

Comment:
Did you check if this page wasn't replaced by and MFE? @sergivalero20

Reply:
As far as I checked, the page replaced by an MFE was **course-home-fragment.html**. I left a comment in the ticket in Jira (https://agile-jira.pearson.com/browse/PADV-509). Anyway I can check again for course_dates_fragment.html.

Reply:
If it is not on the edx platform there is no point in leaving it. @sergivalero20